mdpow/analysis/ensemble.py

```diff
@@ -464,19 +464,34 @@ def _setup_frames(self, trajectory):
         self.times = np.zeros(self.n_frames)
 
     def _single_universe(self):
-        """Calculations on a single Universe object.
-
-            Run on each universe in the ensemble during when
-            self.run in called.
+        """Calculations on a single :class:`MDAnalysis.Universe <MDAnalysis.core.groups.universe.Universe>` object.
+
+           Run on each :class:`MDAnalysis.Universe <MDAnalysis.core.groups.universe.Universe>` 
+           in the :class:`~mdpow.analysis.ensemble.Ensemble` 
+           during when :meth:`run` in called.
+            
+           :exc:`NotImplementedError` will detect whether 
+           :meth:`~EnsembleAnalysis._single_universe`
+           or :meth:`~EnsembleAnalysis._single_frame` 
+           should be implemented, based on which is defined 
+           in the :class:`~mdpow.analysis.ensemble.EnsembleAnalysis`.
```

Comment:
Can you format it so that it fits into 79 character wide lines?

The first line can be broken over multiple lines, there should just be an empty line to separate the summary/header line from the synopsis.